iotilebuild/iotile/build/config/site_scons/pythondist.py

```diff
@@ -219,11 +227,20 @@ def generate_setup_py(target, source, env):
         data['deps'] += tile.support_wheel_depends
 
     data['entry_points'] = entry_points
+    data['include_package_data'] = True
 
     outdir = os.path.dirname(str(target[0]))
 
     render_template('setup.py.tpl', data, out_path=str(target[0]))
 
+    manifest_fn = outdir + '/MANIFEST.in'
+    with open(manifest_fn, 'w') as manifest_file:
```

Comment:
Not sure if this is the preferred way to generate a file.

Reply:
It's fine